spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractMessageChannelBinder.java

```diff
@@ -286,6 +289,63 @@ else if (e instanceof ProvisioningException) {
 		}
 	}
 
+
+	@Override
+	public Binding<PollableSource<MessageHandler>> bindPollableConsumer(String name, String group,
```

Comment:
should it be `doBind...` just to maintain symmetry with other methods?

Reply:
No; this is the interface method; the others are `do...` because they are invoked by the super class. Only binders that implement `PollableConsumerBinder` have this method (which does not include the superclass).